concurrency/3.0/_index.md

```diff
@@ -1,31 +1,46 @@
 ---
-title: "Jakarta Concurrency 3.0 (Under Development)"
-date: 2021-05-20
+title: "Jakarta Concurrency 3.0"
+date: 2022-02-28
 summary: "Jakarta EE 10 Release"
 ---
 Jakarta Concurrency provides a specification for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
 
-# Plan
-Jakarta Concurrency 3.0 will target the Jakarta EE 10 platform release. 
+* [Jakarta Concurrency 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.cu/releases/3.0.0)
+  * [Jakarta EE Platform 10 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee10/JakartaEE10ReleasePlan)
+* [Jakarta Concurrency 3.0 Specification Document](./jakarta-concurrency-spec-3.0.pdf) (PDF)
+* [Jakarta Concurrency 3.0 Specification Document](./jakarta-concurrency-spec-3.0.html) (HTML)
+* [Jakarta Concurrency 3.0 Javadoc](./apidocs)
+* Jakarta Concurrency 3.0 TCK (includes signature tests) [staging](https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-tck/3.0.0/)
+  * [mirrored](https://public.dhe.ibm.com/ibmdl/export/pub/software/olrepo/io/openliberty/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-tck/3.0.0.20220126/jakarta.enterprise.concurrent-tck-3.0.0.20220126.jar)
+  * [SHA](https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-tck/3.0.0/jakarta.enterprise.concurrent-tck-3.0.0.pom.sha1)
+* Maven coordinates
+  * [jakarta.enterprise.concurrent:jakarta.enterprise.concurrent-api:3.0.0](https://search.maven.org/artifact/jakarta.enterprise.concurrent/jakarta.enterprise.concurrent-api/3.0.0/jar) Currently located on [staging](https://jakarta.oss.sonatype.org/content/groups/staging/jakarta/enterprise/concurrent/jakarta.enterprise.concurrent-api/3.0.0/) until release is approved, and then we will push it to Maven.
 
-Jakarta Concurrency 3.0 will aim to contain.
```

Comment:
Sorry about that. I had removed the Plan section because it was absent in the 2.0 release review which I was using as a guide, but now I realize it was a bad idea to copy from that because 2.0 didn't actually have any new function and was just the jakarta package rename.

To correct this, I'm following your first option and restoring the Plan section with a list of new features included just prior to the release information.